core-processor/src/configs.rs

```diff
@@ -53,6 +54,8 @@ pub struct AllocationsConfig {
     pub mem_grow_cost: u64,
     /// Load page cost.
     pub load_page_cost: u64,
+    /// Second load page cost.
```

Comment:
So, if the name is `second_` then we should explain in comment that it's not second only